spec/fixtures/files/sortable/reports_controller.yaml

```diff
@@ -0,0 +1,53 @@
+:entities:
+  - :factory: :v2_report
+    :title: 'aba'
+    :business_objective: 'aba'
+    :compliance_threshold: 90
+    :os_major_version: 9
+    :account: ${account}
+
+  - :factory: :v2_report
+    :title: 'bac'
+    :business_objective: 'bac'
+    :compliance_threshold: 85
+    :os_major_version: 8
+    :account: ${account}
+
+  - :factory: :v2_report
+    :title: 'abc'
+    :business_objective: 'abc'
+    :compliance_threshold: 85
+    :os_major_version: 8
+    :account: ${account}
+
+  - :factory: :v2_report
+    :title: 'bca'
+    :business_objective: 'bca'
+    :compliance_threshold: 80
+    :os_major_version: 7
+    :account: ${account}
+
+  - :factory: :v2_report
+    :title: 'aab'
+    :business_objective: 'aab'
+    :compliance_threshold: 60
+    :os_major_version: 7
+    :account: ${account}
+
+:queries:
+  - :sort_by:
+      - 'title'
+    :result: [4, 0, 2, 1, 3]
+  - :sort_by:
+      - 'business_objective'
+    :result: [4, 0, 2, 1, 3]
+  - :sort_by:
+      - 'compliance_threshold'
+    :result: [4, 3, [1, 2], 0]
+  - :sort_by:
+      - 'os_major_version'
+    :result: [[3, 4], [2, 1], 0]
+  - :sort_by:
+      - 'os_major_version'
+      - 'compliance_threshold'
+    :result: [4, 3, [1, 2], 0]
```

Comment:
Not sure if we need to sort by any system counts, because the frontend is broken ...

Reply:
We need to do an assessment of the compliance `%` for the overall report so we can sort based on that. Probably an arel function that uses the existing fields to compute this number as a third argument of `sortable_by`.